content/en/docs/setup/install/multicluster/common.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2034,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Initialize KUBECONFIG_FILES and KUBE_CONTEXTS
+_set_kube_vars
+
+# set_single_network_vars initializes all variables for a single network config.
+function set_single_network_vars
+{
+  export CTX_CLUSTER1="${KUBE_CONTEXTS[0]}"
+  export CTX_CLUSTER2="${KUBE_CONTEXTS[1]}"
+}
+
+# set_multi_network_vars initializes all variables for a multi-network config.
+function set_multi_network_vars
+{
+  export CTX_CLUSTER1="${KUBE_CONTEXTS[0]}"
+  export CTX_CLUSTER2="${KUBE_CONTEXTS[2]}"
+}
+
+# delete_namespaces removes the istio-system and sample namespaces on both
+# CLUSTER1 and CLUSTER2.
+function delete_namespaces()
+{
+  # Run the delete on both clusters concurrently
+  _delete_namespaces_cluster1 &
+  _delete_namespaces_cluster2 &
+  wait
+}
+
+# _delete_namespaces_cluster1 removes the istio-system and sample namespaces on both
+# CLUSTER1.
+function _delete_namespaces_cluster1()
+{
+  kubectl delete ns istio-system sample --context="${CTX_CLUSTER1}" --ignore-not-found
+}
+
+# _delete_namespaces_cluster2 removes the istio-system and sample namespaces on both
+# CLUSTER2.
+function _delete_namespaces_cluster2()
+{
+  kubectl delete ns istio-system sample --context="${CTX_CLUSTER2}" --ignore-not-found
+}
```

Comment:
I wonder if this bunch is even more common than for just this doc's tests, and could be moved up to tests/util. WDYT?

Reply:
I don't think so. It relies specifically on the environment variables `CLUSTER1` and `CLUSTER2`